lib/solidus_stripe/configuration.rb

```diff
@@ -15,9 +16,15 @@ class Configuration
     #  its creation. Defaults to `Stripe::Webhook::DEFAULT_TOLERANCE`.
     attr_accessor :webhook_signature_tolerance
 
+    # @!attribute [rw] refund_reason_name
+    #   @return [String] `Spree::RefundReason#name` used for Stripe-generated
```

Comment:
Too much indent here?

Reply:
I think it's ok. See https://rubydoc.info/gems/yard/file/docs/Tags.md#attribute

Reply:
Ah, because it refers to the attribute. Good to know, thanks!